src/BlazorUI/Bit.BlazorUI/Components/Utilities/SwipeTrap/BitSwipeTrap.ts

```
@@ -0,0 +1,123 @@
+            SwipeTrap._swipeTraps.push(swipeTrap);
+        }
+
+        public static dispose(id: string) {
+            const swipeTrap = SwipeTrap._swipeTraps.find(r => r.id === id);
+            if (!swipeTrap) return;
+
+            SwipeTrap._swipeTraps = SwipeTrap._swipeTraps.filter(r => r.id !== id);
+            swipeTrap.dispose();
+        }
+    }
+
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)